src/html/Options.html

```diff
@@ -84,5 +84,6 @@ <h3 data-i18n="local_file_saving"></h3>
 
       <script src='/js/extra-permission.js'></script>
       <script src='/js/i18n.js'></script>
+      <script src='/js/Options.js'></script>

```

Comment:
The JS needs to be loaded here.

Reply:
Effectivly, i don't remember me why i removed the Options.js file loading in my html file....